CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ Changed:
 - APIs accepting a `FileSystem` and `Path` now have the `FileSystem` coming before the `Path` in the parameter list. Compatibility functions are retained for this version, but will be removed in the next version.
 
 Fixed:
+- Fix memory leaks caused by reference cycles on iOS. We got into trouble mixing garbage-collected Kotlin objects with reference-counted Swift objects.
```

Comment:
Can you do a quick rebase? I released these notes a few hours ago and there's a new "Unreleased" section now.

Reply:
Oooh yep. Done.